.github/workflows/release.yaml

```diff
@@ -86,3 +86,43 @@ jobs:
           asset_path: ${{ env.ASSET }}
           asset_name: ${{ env.ASSET }}
           asset_content_type: application/gzip
+  publish-docker:
+    name: publish-docker
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Authenticate with github container registry
+        run: echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
@lpil You will need to create a Personal Access Token and set it as a repository secret

https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token